server/sonar-server/src/main/java/org/sonar/server/ce/ws/ActivityAction.java

```diff
@@ -160,19 +158,19 @@ public void handle(Request wsRequest, Response wsResponse) throws Exception {
   }
```

Comment:
given this commit's description, I would have expected a change in the description of the param `PARAM_COMPONENT_ID`